### PR TITLE
Improved keyboard accessibility of the find-replace popup

### LINKED ISSUE #1699: Improve keyboard accessibility of the find-replace popup
#### Nature of issue?

- Existing feature enhancement

#### Feature enhancement details:
Hello @catarak, sorry for being AFK for quite a while. Since the find & replace feature has been released, I think the order in which the input fields on the popup are accessed on Tab presses can be changed for better UX. I would like to work on this.

### PR DESCRIPTION
**Pull request by SundeepChand**
Fixes #1699 
I have verified that this pull request:

* [x] has no linting errors (`npm run lint`)
* [x] is from a uniquely-named feature branch and has been rebased on top of the latest `develop` branch. (If I was asked to make more changes, I have made sure to rebase onto `develop` then too)
* [x] is descriptively named and links to an issue number, i.e. `Fixes #123`

Hello @catarak. Please review my work and please verify the Spanish & Japanese translations of 'Toggle Replace'.

### CODE REVIEW

**Comment by catarak** [comment]
@koji can you verify the Japanese translation?

@oruburos can you verify the Spanish translation? 

Thank you!!

**Comment by oruburos** [comment]
@catarak  All good in the Spanish translation.

**Comment by koji** [comment]
@catarak I would recommend this, `置換の切り替え` instead of the current `トグル置換`.
@SundeepChand, could you change this ↑, please?

By the way, thank you for the release, one person told me that the translation has been released.
https://twitter.com/takawo/status/1329613670188367872

**Comment by SundeepChand** [comment]
Hello @koji, Thanks for your suggestions. I've updated the same.

**Comment by catarak** [comment]
Thanks for adding those changes @SundeepChand! I was testing the JS changes that you made, and I noticed that now when I click on the "Replace" button, the replace doesn't happen, but it does work when I click "Replace All".

**Comment by SundeepChand** [comment]
Thanks @catarak for pointing this out, this bug is there in production as well. Its fixed now.

File: client/styles/components/_editor.scss [hunk 1]
```diff
@@ -121,7 +121,7 @@ pre.CodeMirror-line {
   height: 100%;
 }
 
-.CodeMirror-find-div {
+.CodMirror-search-inputs {
```
Comment (catarak) [inline client/styles/components/_editor.scss]: i'm assuming you mean for this class to be `.CodeMirror-search-inputs`? I don't see anything in the HTML that has `.CodMirror-search-inputs`, so maybe you can just delete this CSS?

**Review by catarak — CHANGES_REQUESTED** [review]

File: client/styles/components/_editor.scss (hunk 1 above)
Comment (SundeepChand) [reply client/styles/components/_editor.scss]: I'm sorry. A mistake on my side.

Comment (catarak) [reply client/styles/components/_editor.scss]: no worries!!

**Review by catarak — APPROVED** [review]

**Pull request merged**